pkg/uefivars/guid.go

```diff
@@ -0,0 +1,33 @@
+// Copyright 2015-2020 the u-root Authors. All rights reserved
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+//
+// SPDX-License-Identifier: BSD-3-Clause
+//
+
+package uefivars
+
+import (
+	"github.com/google/uuid"
+)
+
+//A mixed-endianness guid, as used by MS and UEFI. For normal uuid-related functionality, use a  package like github.com/google/uuid .
```

Comment:
just a nit, but golint wouldn't like a lot of these comments because they don't begin with the name of the thing:

```go
// MixedGUID is a mixed-endianness GUID as used by UEFI.
type MixedGUID [16]byte
```

also this is one where we could s/MixedGuid/MixedGUID/ per https://github.com/golang/go/wiki/CodeReviewComments#initialisms

Reply:
Fixed a bunch of these - may have missed some though.